src/import/load-sound.js

```diff
@@ -61,6 +95,12 @@ const loadSound = function (sound, runtime, soundBank) {
         runtime.storage.load(runtime.storage.AssetType.Sound, md5, ext)
     ).then(soundAsset => {
```

Comment:
Actually, thinking about this further: we should probably have tests that verify `loadCostume` and `loadSound` return default assets if `storage.load` rejects. That might be out of scope for the GQM project, though, as long as what happens now is anything other than a data-loss-by-default-asset situation. (In this case I think it might be an unhandled exception => BSOD?)

Reply:
I decided to leave the error case out so as not to complicate this PR (which was supposed to be focused on vector costumes) too much more. This is similar to how I left off adding a `.catch` to `loadBitmap_` for a later PR as well.

I think there should be a separate PR for broken bitmap costumes and a separate PR for corrupted sounds (or general sound error handling). I just wanted to make sure that we are handling storage.load resolving null in the same PR where we're bringing in the update to storage 2.0.1.

I am hoping to get those other 2 PRs in soon (barring complications I run into testing bitmaps..)

Reply:
Nvm! I *will* address corrupted sound stuff in this PR, since an error thrown here could result in the editor crashing, which might be more likely now that storage is swallowing less errors.

Also I assume you're talking about having unit tests for loadCostume and loadSound in addition to the integration tests? I might add those after this PR.

Reply:
Added https://github.com/LLK/scratch-vm/pull/3637/commits/8ab21dd701a6c2b20260c30cd4994a2e4eb75900 to address the sound error handling stuff